packages/adapters/subgraph/src/lib/operations/queries.ts

```diff
@@ -829,3 +868,62 @@ export const getRootManagerMetaQuery = (domain: string) => {
     }
   `;
 };
+
+export const getStableSwapPoolsQuery = (domain: string) => {
+  const { config } = getContext();
+  const prefix = config.sources[domain].prefix;
+  const queryString = `
+  ${prefix}_stableSwaps ( 
+    first: 100
```

Comment:
In current state it would retrieve only the first hundred every time.
Is there a scenario where there are more than 100 ?

Reply:
`first: 1000` will be pretty enough!